granulate_utils/metadata/databricks_client.py

```diff
@@ -43,46 +50,80 @@ def get_webui_address() -> Optional[str]:
 
     def get_job_name(self) -> Optional[str]:
         # Retry in case of a connection error, as the metrics server might not be up yet.
-        for i in range(MAX_RETRIES):
-            time.sleep(10)
+        retries = 0
+        while retries < MAX_RETRIES:
```

Comment:
As discussed, please refactor this look to run until the "timeout" (2 minutes) passes, and not for N retries, as the retries themselves can take time.

Reply:
a0696c4

Reply:
Yes, that's good.